### PR TITLE
Chore/first release parameters

### PR DESCRIPTION
**Pull request by osmaczko**
closes: #28

### CODE REVIEW

File: packages/DApp/src/components/Rules.tsx [hunk 1]
```diff
@@ -17,45 +17,52 @@ export function Rules() {
...
       <Rule>
         <RuleIcon>🗑️</RuleIcon>
         <RuleText>
           If a vote to remove a community ends in favour of the removal, the community will disappear from the
-          directory. Otherwise, a new vote for removal can be submitted after about 30 days.
+          directory. Otherwise, a new vote for removal can be submitted after about 2 days.
         </RuleText>
       </Rule>
-      <Rule>
+      {/* <Rule>
```
Comment (osmaczko) [inline packages/DApp/src/components/Rules.tsx:30]: todo: create a feature request for that

File: packages/DApp/src/components/Rules.tsx [hunk 2]
```diff
@@ -17,45 +17,52 @@ export function Rules() {
...
       <Rule>
         <RuleIcon>⏳</RuleIcon>
         <RuleText>
-          Each vote lasts for two weeks. When the voting period ends, someone has to finalize the vote with a
+          Each verification period lasts for one day. When the verification ends, someone has to finalize the vote with a
           finalization transaction.
         </RuleText>
       </Rule>
-      <Rule>
+      {/* <Rule>
```
Comment (osmaczko) [inline packages/DApp/src/components/Rules.tsx:51]: todo: create a feature request for that

File: packages/DApp/src/components/Rules.tsx [hunk 3]
```diff
@@ -17,45 +17,52 @@ export function Rules() {
...
         <RuleIcon>⭐</RuleIcon>
         <RuleText>
-          There is an always ongoing vote for which communities from the directory should appear in the Weekly Featured
-          section. Every week, five communities with the most votes are added to the section, replacing the five
-          communities that were featured the previous week. After the community leaves Weekly Featured, it cannot get
-          there again for three weeks.
+          There is an always ongoing vote for which communities from the directory should appear in the Featured
+          section. Every three days, three communities with the most votes are added to the section, replacing the three
+          communities that were featured previously. After the community leaves Featured section, it cannot get
+          there again for around a week.
```
Comment (felicio) [inline packages/DApp/src/components/Rules.tsx:65]: "around a week"? we cannot be specific?

**Review by felicio — APPROVED** [review]

File: packages/DApp/src/components/Rules.tsx (hunk 3 above)
Comment (osmaczko) [reply packages/DApp/src/components/Rules.tsx]: Not really, we don't have control over when the vote is started and when it is finalized, as it is the manual action required by peers.

Comment (felicio) [reply packages/DApp/src/components/Rules.tsx]: > After the community leaves Featured section, it cannot get
          there again for around a week.

Where is this evaluated in the code?

Comment (osmaczko) [reply packages/DApp/src/components/Rules.tsx]: It is the contract that validates it: https://github.com/status-im/community-dapp/blob/30e8b267d67379d0eb2bc6da598615f5586baf63/packages/contracts/contracts/FeaturedVotingContract.sol#L185-L196

**Pull request merged**